src/lib/rest/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ SET (SOURCES
     orionReply.cpp
     OrionError.cpp
     HttpStatusCode.cpp
+    curl_context.cpp
```

Comment:
The files that contain one C++ class normally have the same name as the class.
Classes start with uppercase so those files also start with uppercase.
Other files contain collections of stuff (e.g. sem.cpp - semaphore stuff) so that file starts with lowercase.
Messy? Not really. I think it is pretty good actually. I wouldn't change a thing.